core/xml.js

```diff
@@ -334,9 +338,68 @@ Blockly.Xml.domToWorkspace = function(xml, workspace) {
   if (workspace.setResizesEnabled) {
     workspace.setResizesEnabled(true);
   }
+  return newblockids;
 };
 
 /**
+ * Decode an XML DOM and create blocks on the workspace.
+ * Append the new blocks at the bottom of the existing blocks
```

Comment:
"Positions the new blocks immediately below any existing blocks, such that the left-most blocks are aligned."